src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -639,9 +641,8 @@ public static void annotateReadLikelihoodsWithSupportedAlleles(final VariantCont
             }
 
             // keep track of the haplotypes that contain this particular alternate allele
-            final Allele alt = call.getAlternateAllele(0);
-            final Predicate<VariantContext> hasThisAlt = vc -> (vc.getStart() == call.getStart() && vc.getAlternateAlleles().contains(alt)) ||
-                    (Allele.SPAN_DEL.equals(alt) && vc.getStart() < call.getStart() && vc.getEnd() >= call.getStart());
+            final Allele alt = getSiteSpecificAlternateAllele(call);
```

Comment:
If we're going to continue to handle only the case of a single alt allele, then we should modify the code a few lines up to test not just `if ( ! isBiallelic(call) )` but rather test for biallelic sites where one allele is ref.

Reply:
The `isBiallelic` method for this class was already testing for that condition (but wasn't aware of * alleles). I've renamed the method to make it clear that it's explicitly testing for the case where the site is biallelic with a single site-specific alt allele.